scrapy/http/response/json.py

```diff
@@ -0,0 +1,16 @@
+"""
+This module implements the JsonResponse class which adds encoding
+discovering through JSON encoding declarations to the TextResponse class.
+
+See documentation in docs/topics/request-response.rst
+"""
+
+import json
```

Comment:
Could you please add a blank line between these imports, as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports)?